tgui/packages/tgui/interfaces/AntagInfoHeretic.tsx

```diff
@@ -35,20 +35,33 @@ const hereticYellow = {
   color: 'yellow',
 };
 
+type IconParams = {
+  icon: string;
+  state: string;
+  frame: number;
+  dir: number;
+  moving: boolean;
+};
+
 type Knowledge = {
   path: string;
+  icon_params: IconParams;
   name: string;
   desc: string;
   gainFlavor: string;
   cost: number;
   disabled: boolean;
-  hereticPath: string;
   color: CssColor;
+  finished: boolean;
+  ascension: boolean;
```

Comment:
If these are from dm it's BooleanLike

Reply:
DmIcon's `movement` parameter seems to complain when passed a BooleanLike... Should I do it anyway?

Reply:
hmm maybe cast it with `!!` ?

Reply:
i could be wrong here, not sure if the query should also reflect `0 | 1` rather than T/F. if it *is* expecting a 0/1, then the type on DmIcon is incorrect and you can ignore this